api/pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>de.valtech.aecu</groupId>
         <artifactId>aecu</artifactId>
-        <version>6.4.1-SNAPSHOT</version>
+        <version>6.5.1-SNAPSHOT</version>
```

Comment:
also there is a build error due to wrongly referenced versions. you can use `mvn versions:set -DnewVersion=6.5.0-SNAPSHOT -DgenerateBackupPoms=false`  on the root folder in case